document/js-api/index.bs

```diff
@@ -760,7 +766,7 @@ Each {{Table}} object has a \[[Table]] internal slot, which is a [=table address
     The <dfn constructor for="Table">Table(|descriptor|, |value|)</dfn> constructor, when invoked, performs the following steps:
     1. Let |elementType| be [=ToValueType=](|descriptor|["element"]).
     1. If |elementType| is not a [=reftype=],
-        1. [=Throw=] a {{TypeError}} exception.
+        1. Throw a {{TypeError}} exception.
```

Comment:
Why this change?

Reply:
Using this annotation around Throw makes it point to the definition of throwing a wasm exception (https://webassembly.github.io/exception-handling/js-api/index.html#webassembly-throw) but this step is actually just meant to throw a regular JS exception. Note how the other places where this spec says to throw JS exceptions for error conditions just use the word "throw" with no links.